Resources/Textures/Tiles/attributions.yml

```diff
@@ -95,3 +95,8 @@
   license: "CC-BY-SA-3.0"
   copyright: "taken at https://github.com/ParadiseSS13/Paradise/blob/8b7f4c8b69c74c6de5a755272eb8d3520f3d87c7/icons/turf/floors.dmi"
   source: "https://github.com/ParadiseSS13/Paradise"
+  
+- files: ["chromite.png"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "taken at commit 0587dd16e28108bdf0b0a28e2caae4319845e861, and recolored by TheShuEd"
+  source: "https://github.com/Mojave-Sun/mojave-sun-13"
```

Comment:
Is this the right commit you used?

Reply:
link copied from the original texture in ss14 
then recolored